tests/unit/__init__.py

```diff
@@ -1627,5 +1634,158 @@ def setUp(self):
+            self.app.router['SupportCentreListCentres:get'].url_for(
+                display_region='en', postcode=self.postcode_valid)
+        self.get_support_centre_list_of_centres_postcode_valid_cy = \
+            self.app.router['SupportCentreListCentres:get'].url_for(
+                display_region='cy', postcode=self.postcode_valid)
+
+        # Contents
+
+        self.content_support_centre_enter_postcode_title_en = 'Find a support centre'
+        self.content_support_centre_enter_postcode_secondary_en = \
+            'To find your nearest support centre, we need your postcode.'
+        self.content_support_centre_enter_postcode_error_empty_en = 'You have not entered a postcode'
+        self.content_support_centre_enter_postcode_error_invalid_en = 'The postcode is not a valid UK postcode'
+        # TODO Add Welsh Translation
+        self.content_support_centre_enter_postcode_title_cy = 'Find a support centre'
+        # TODO Add Welsh Translation
+        self.content_support_centre_enter_postcode_secondary_cy = \
+            'To find your nearest support centre, we need your postcode.'
+        # TODO Add Welsh Translation
+        self.content_support_centre_enter_postcode_error_empty_cy = 'You have not entered a postcode'
+        # TODO Add Welsh Translation
+        self.content_support_centre_enter_postcode_error_invalid_cy = 'The postcode is not a valid UK postcode'
+
+        self.content_support_centre_list_of_centres_result_one_google_url = \
+            'https://www.google.com/maps/search/?api=1&query=53.380582,-1.466986'
+
+        self.content_support_centre_list_of_centres_title_en = 'Support centres near ' + self.postcode_valid
+        self.content_support_centre_list_of_centres_result_one_location_name_en = 'Sheffield Central Library'
+        self.content_support_centre_list_of_centres_result_two_location_name_en = 'University of Sheffield'
+        self.content_support_centre_list_of_centres_result_one_distance_away_en = \
+            '<span class="u-mb-s tag">1.3 miles away</span>'
+        self.content_support_centre_list_of_centres_result_one_address_en = \
+            '<p>Surrey Street<br> Sheffield<br>S1 1XZ</p>'
+        self.content_support_centre_list_of_centres_result_one_telephone_en = \
+            'Telephone: <span>+44 (0)11 4273 4712</span>'
+        self.content_support_centre_list_of_centres_result_one_email_en = \
+            'Email: <a href="mailto:test@email.com">test@email.com</a>'
+        self.content_support_centre_list_of_centres_result_one_times_monday_en = 'Monday &ndash;&nbsp;10:30am to 5:15pm'
+        self.content_support_centre_list_of_centres_result_one_times_tuesday_en = 'Tuesday &ndash;&nbsp;10am to 5pm'
+        self.content_support_centre_list_of_centres_result_one_times_sunday_en = 'Sunday &ndash;&nbsp;Closed'
+        self.content_support_centre_list_of_centres_result_one_times_census_saturday_en = \
+            'Census Saturday, 20 March &ndash;&nbsp;10am to 4pm'
```

Comment:
maybe add may bank holiday and others in here ?

Reply:
Now includes values for open and closed for all days